tapo/src/requests/set_device_info/trv.rs

```diff
@@ -0,0 +1,79 @@
+use serde::Serialize;
+
+use crate::error::Error;
+
+#[derive(Debug, Default, Serialize)]
+pub(crate) struct TrvSetDeviceInfoParams {
```

Comment:
The fields inside the struct shouldn't need to be `pub`.

Reply:
Implemented in 0dead6c.